pyinterprod/pronto/goa.py

```diff
@@ -234,3 +234,82 @@ def _get_constraints(term_id: str, ancestors: dict, constraints: dict) -> set:
         result |= _get_constraints(parent_id, ancestors, constraints)
 
     return result
+
+
+def import_go_constraints(go_url: str, pg_url: str):
+    pg_con = psycopg.connect(**url2dict(pg_url))
+    ora_con = oracledb.connect(go_url)
+    ora_cur = ora_con.cursor()
+
+    ora_cur.execute(
+        """
+        SELECT UNION_ID, LISTAGG(TAXON_ID, ',')
+        FROM GO.NCBI_TAXON_UNIONS
+        GROUP BY UNION_ID
+        """
+    )
+    union2taxon = {}
+    for union_id, taxon_ids in ora_cur:
+        union2taxon[union_id] = taxon_ids.split(",")
+
+    ora_cur.execute(
+        """
+        SELECT T.GO_ID, AT.GO_ID, TC.RELATIONSHIP, TC.TAX_ID_TYPE, TC.TAX_ID
```

Comment:
No need to select columns we don't use:

```suggestion
        SELECT T.GO_ID, TC.RELATIONSHIP, TC.TAX_ID_TYPE, TC.TAX_ID
```